code/script.js

```diff
@@ -1,19 +1,133 @@
-// Start here
+//Slows down the first alert to make everything on the site have time to load before the alert appears
+window.onload = function () { 
+  setTimeout(function() {
 
 // Step 1 - Welcome and introduction
-// Your code goes here
+//An alert with a welcoming message
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  `Welcome to our Pizzeria The Cheesy Slice! Ready to Start? - Click 'OK' to begin.`
+)
+
+//Using the prompt method to ask for the name of the customer and an alert uses that variable to greet the customer
+const guestName = prompt(`Dear guest, what is your name?`)
+console.log(guestName)
+
+alert(
+  `Hi ${guestName}! We will happily take your order!`
 )
 
 // Step 2 - Food choice
-// Your code goes here
+//Ask the customer for what type of food they want and confirm what they selected in an alert
+const foodChoice = parseInt(
+  prompt(
+    `What are you craving today?
+    1 - Pizza
+    2 - Pasta
+    3 - Salad
+    Please choose the number of what you would like to order.`
+  )
+)
+console.log(foodChoice)
+
+let selectedFood = ""
+
+if (foodChoice === 1) {
+selectedFood = "pizza"
+} else if (foodChoice === 2) {
+  selectedFood = "pasta"
+} else if (foodChoice === 3) {
+  selectedFood = "salad"
+} else {
+  alert(`We can't identify what type of food you want to order. Please choose a valid number.`)
+  process.exit(1)
+}
+
+alert(
+  `You have chosen to order a delicious ${selectedFood}!`
+)
 
 // Step 3 - Subtype choice
-// Your code goes here
+//Using the conditional statement switch method for the customer to choose what dish they want depending on what food they selected previously
+let subMenu = ""
+let dishName = ""
+let selectedDish = ""
+
+switch (selectedFood) {
+  case "pizza": 
+    subMenu = prompt(`What dish would you like to have?
+  1 - Vesuvio
+  2 - Capricciosa
+  3 - Veggie Delight
+  Please choose the number of what you would like to order.`)
+    dishName = ["Vesuvio", "Capricciosa", "Veggie Delight"]
+    break
+  case "pasta": 
+    subMenu = prompt(`What dish would you like to have?
+  1 - Bolognese
+  2 - Carbonara
+  3 - Frutti di Mare
+  Please choose the number of what you would like to order.`)
+    dishName = ["Bolognese", "Carbonara", "Frutti di Mare"]
+    break
+  case "salad": 
+    subMenu = prompt(`What dish would you like to have?
+  1 - Caesar salad
+  2 - Greek salad
+  3 - Cobb salad
+  Please choose the number of what you would like to order.`)
+    dishName = ["Caesar salad", "Greek salad", "Cobb salad"]
+    break
+  default:
+    alert(`We can't identify what dish you want to order. Please choose a valid number.`)
+    process.exit(1)
+}
+
+selectedDish = dishName[subMenu - 1] 
+
+console.log(dishName)
+
+alert(
+  `You have chosen to order a yummy ${selectedDish}!`
+)
 
 // Step 4 - Age
-// Your code goes here
+//Using a prompt to ask for the age of the customer in order to decide a price
+const guestAge = parseInt(
```

Comment:
It's great to see you frequently using the parseInt-method, inorder to make the computer understand it's a number not a string the customer has entered! :)